src/labels/CustomLabels.labels

```diff
@@ -6027,6 +6027,22 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>errErrorUnsupportedDMLType</shortDescription>
         <value>Unsupported Operation: Unable to log an Error for this type of DML operation.</value>
     </labels>
+    <labels>
+        <fullName>euExploreTrailheadButton</fullName>
```

Comment:
Since this is part of the get start page, let's keep with the existing "gs" prefix on all labels, pages, classes, etc.

Reply:
I thought about it, but as this is another start page, I prefered having a separated prefix to keep searching in the future for labels easier(less in the same prefix is better in my opinion)

Reply:
I'm ok, with a different prefix, but I don't think it should be just "eu". I'd prefer "gseu" this way it still groups with all other get started page resources.

Reply:
done